content/issues/trying-out-the-new-projects-experience/creating-a-project.md

```diff
@@ -81,11 +81,11 @@ In board layout:
 You can archive an item to keep the context about the item in the project but remove it from the project views. You can delete an item to remove it from the project entirely.
 
 1. Select the item(s) to archive or delete. To select multiple items, do one of the following:
-     - `cmd + click` (Mac) or `ctrl + click` (Windows/Linux) each item.
-     - Select an item then `shift + arrow-up` or `shift + arrow-down` to select additional items above or below the initially selected item.
-     - Select an item then `shift + click` another item to select all items between the two items.
-     - Enter `cmd + a` (Mac) or `ctrl + a` (Windows/Linux) to select all items in a column in a board layout or all items in a table layout.
-2. To archive all selected items, enter `e`. To delete all selected items, enter `del`. Alternatively, select the {% octicon "triangle-down" aria-label="the item menu" %} (in table layout) or the {% octicon "kebab-horizontal" aria-label="the item menu" %} (in board layout), then select the desired action.
+     - <kbd>Command</kbd>+Click (Mac) or <kbd>Ctrl</kbd>+Click (Windows/Linux) each item.
+     - Select an item then <kbd>Shift</kbd>+<kbd>↑</kbd> or <kbd>Shift</kbd>+<kbd>↓</kbd> to select additional items above or below the initially selected item.
```

Comment:
@AnilSeervi Please use arrow key names instead of symbols. In this case, please use `Down arrow key`. You can refer [Key names guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts#key-names) for more information on how to use arrow keys.

Reply:
I was just following [guidelines](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#keyboard-shortcuts) that were described in the link you provided, where it says

- Use symbols for keys of special character, not full words. - **Use:** `.`, `,`, and `→`. - **Avoid:** `Period`, `Comma`, and `Right arrow`.

What should be done here?

Reply:
So sorry for the confusion! I double checked the guidelines and what you have here is correct. I will go ahead approve the changes!! Thanks for all your awesome work 💖

Reply:
> So sorry for the confusion! I double checked the guidelines and what you have here is correct.

No Problem.

> I will go ahead approve the changes!! Thanks for all your awesome work 💖 

Happy to help <3